### ISSUE TITLE
Video tutorial for setting up the crawler 

### ISSUE THREAD

**Issue by franciscawijaya** [privacy-tech-lab/gpc-web-crawler#106]
After we finish setting up the docker, we can create a video tutorial to help others set up the crawler.

**Comment by SebastianZimmeck** [comment]
This issue is now best tackled by @Mattm27 once the Docker setup (#98) is implemented.

**Comment by SebastianZimmeck** [comment]
Once the Dockerization (#138) and release (#76) is complete, hopefully Monday next week, @Mattm27 and @eakubilo will create a setup video for the readme.

**Comment by avan36** [comment]
I made an initial demo video here let me know if you have feedback on it or things to change!

https://github.com/user-attachments/assets/2febb535-57f3-4c56-ba7a-e1d2698872c5

**Comment by SebastianZimmeck** [comment]
@RubyFri and @sagealtman will be working with @avan36 on the tutorial.

**Comment by SebastianZimmeck** [comment]
@RubyFri explained that [@avan36's tutorial video](https://github.com/privacy-tech-lab/gpc-web-crawler/issues/106#issuecomment-2743416247) works well. 

@RubyFri will polish it a bit, e.g., with privacy-tech-lab branding and end credits.

Then, we can link it in the [readme at the beginning of the Development section](https://github.com/privacy-tech-lab/gpc-web-crawler?tab=readme-ov-file#4-development).

Here are two example videos: [How to Implement Global Privacy Control? A Guide for Web Publishers.](https://www.youtube.com/watch?v=zfiyNqE72lk&t=31s) and [How to Opt Out from Ad Tracking with Global Privacy Control? A Guide for Web Users.](https://www.youtube.com/watch?v=k361rJn-JAE&t=2s).

Also, we can post it [like in the OptMeowt repo](https://github.com/privacy-tech-lab/gpc-optmeowt?tab=readme-ov-file#2-promo-video). Let us test in Firefox as well because sometimes it seems that the image is not displayed correctly. Maybe, we need a screenshot that works with link.

Let us also make sure that the setup instructions are aligned with what is said in the video. If not, adapt.

**Comment by RubyFri** [comment]
I made a more formal version of @avan36 's video. This version has privacy-tech-lab branding and does not show the other windows open on the laptop.
> post it [like in the OptMeowt repo](https://github.com/privacy-tech-lab/gpc-optmeowt?tab=readme-ov-file#2-promo-video).

I will do the above once at least one person looks it over and confirms that it is good!


https://github.com/user-attachments/assets/d2a26381-9dd9-4b7a-9352-e4e4806593ae

**Comment by sagealtman** [comment]
I followed the video's steps to attempt to set up the crawler on my personal laptop (a MacBook Pro running Sequoia 15.6) and got the following error message when trying to run the crawler:

`RUN apt-get update && apt-get install -y curl mariadb-client && rm -rf /var/lib/apt/lists/*

Ign:1 http://deb.debian.org/debian buster InRelease
Ign:2 http://deb.debian.org/debian-security buster/updates InRelease
Ign:3 http://deb.debian.org/debian buster-updates InRelease
Err:4 http://deb.debian.org/debian buster Release
404  Not Found [IP: 151.101.46.132 80]
Err:5 http://deb.debian.org/debian-security buster/updates Release
404  Not Found [IP: 151.101.46.132 80]
Err:6 http://deb.debian.org/debian buster-updates Release
404  Not Found [IP: 151.101.46.132 80]
Reading package lists...
E: The repository 'http://deb.debian.org/debian buster Release' does not have a Release file.
E: The repository 'http://deb.debian.org/debian-security buster/updates Release' does not have a Release file.
E: The repository 'http://deb.debian.org/debian buster-updates Release' does not have a Release file. `

Docker identifies it as an issue with the setup for the RestAPI container, and from the error message it looks like the setup is struggling to find the necessary Debian release. My laptop is fairly new, so this might be also be a compatibility issue. Otherwise, the video looks good.

**Comment by sagealtman** [comment]
Doing some digging, it looks like the buster version of Debian is not listed at http://deb.debian.org/debian. This may be due to the release of Debian 13.0 (trixie) recently, which may have led an older version like buster to have been moved to the archives.

**Comment by SebastianZimmeck** [comment]
Thank you, @sagealtman and @RubyFri! We may need a new issue on this. We can discuss later today.

**Comment by SebastianZimmeck** [comment]
OK, either of the first two options would work well.

**Comment by Spongebosch** [comment]
https://github.com/user-attachments/assets/e7f49f64-1d73-4a7d-add8-73dd576e726c
Here's the edited version.

It looks like it is embedded correctly. The URL provided for the video is a little different from the one in Privacy Pioneer, and I wasn't able to find a way to get it into the same github.com/org/repo/assets/... format, but I believe the github.com/user-attachments/assets/... version should work just fine.
